packages/bsky/tests/admin/get-record.test.ts

```diff
@@ -69,15 +64,15 @@ describe('admin get record view', () => {
     })
   })
 
-  it('gets a record by uri, even when taken down.', async () => {
+  it.skip('gets a record by uri, even when taken down.', async () => {
```

Comment:
I can't account for why this issue would turn-up all of a sudden, but try adding a `await network.processAll()` after the `basicSeed()` call above.  That ensures that writes have propagated from the pds through the appview—I would expect it to already have been there!

Reply:
goddamn that did the trick!